examples/battery_storage_parallel.jl

```diff
@@ -77,7 +77,7 @@ println("library loaded")
     end
 
     function constraint(t, x, u, xi)
-    	return( (x[1] <= s_bounds[1][2] )&(x[1] >= s_bounds[1][1]))
+    	return true
```

Comment:
Yeah that's kind of what we do if we convert a LinearCostSPModel into StochDynProgModel. I could overload again the constructor but maybe it's time to harmonize the SPmodels constructor as there are almost identical. We already talked about that